packages/base/boolean.gts

```diff
@@ -41,6 +42,10 @@ export default class BooleanField extends FieldDef {
     return Boolean(val);
   }
 
+  static get [emptyValue]() {
+    return false;
+  }
+
```

Comment:
let's make sure to add a test for `[emptyValue]`. we can add this in the `card-basics-test` in the host tests. we should test for Boolean fields and we should test any other field that has the `BaseDef`'s `undefined` empty value.

Reply:
Do we want boolean fields to return `false` by default?

Reply:
From the [`[serialize]`](https://github.com/cardstack/boxel/blob/578cced5fdf5416173c0dd9c956dd726d9d875ca/packages/base/boolean.gts#L40) and the [`[deserialize]`](https://github.com/cardstack/boxel/blob/578cced5fdf5416173c0dd9c956dd726d9d875ca/packages/base/boolean.gts#L44) function in BooleanField, it seems we try to return false if the value null or undefined.

Reply:
what I was wondering was, do we want to have boolean field serialized as `null` or `undefined` if it has no value. if people are agreeing on `false` as default, that's fine with me 👍